csm/core/services/rgw/s3/utils.py

```diff
@@ -37,10 +41,22 @@ def get_rgw_connection_config():
             const.CSM_GLOBAL_INDEX, const.RGW_S3_IAM_ADMIN_USER)
         rgw_connection_config.auth_user_access_key = Conf.get(
             const.CSM_GLOBAL_INDEX, const.RGW_S3_IAM_ACCESS_KEY)
-        rgw_connection_config.auth_user_secret_key = Conf.get(
-            const.CSM_GLOBAL_INDEX, const.RGW_S3_IAM_SECRET_KEY)
+        rgw_connection_config.auth_user_secret_key = \
```

Comment:
Move this logic to service class

Reply:
we can move this to service class at the time of implementing  secret decryption failover scenario. Let me know on this.